apps/devhub/tests/test_views.py

```diff
@@ -1134,6 +1134,7 @@ def test_agreement_first(self):
         assert mock_submit.called
 
     def test_agreement_second(self):
+        self.create_switch('api-key', db=True)
```

Comment:
put this in setUp() instead because it's needed for all API agreement tests. BTW, `TestAgreement` should probably be `TestAPIAgreement`